solutionbox/inception/datalab_solutions/inception/_package.py

```diff
@@ -36,51 +36,106 @@
 
```

Comment:
file level comment: ah, I think my questions about gcloud/subprocess should go in this file. So for structured data, I have a preprocess.py file and a trainer folder so far. Everything is built like the iris sample: designed to work on the command line. So then as a package author, I would make a _package.py file that has all these local_preprocess, cloud_preprocess, local_train, etc functions. I would have these functions simply make a subprocess command like if I was doing everything on the command line. 

I think this is interesting to see how you designed your code for this datalab interface, while mine is designed for the gcloud usage. This is good.